tutorials/get-started/index.md

```diff
@@ -0,0 +1,989 @@
+VSCode and JetBrains products.
+
+See [Mit's new setup page]() for information on how to set up
+your environment and create your first simple Flutter app.
+
+</div>
+
+## Step 1: Create the starting Flutter app
+
+Create a basic Flutter app, using the instructions in
+[Mit’s new page, preferably linked to a specific anchor for
+creating the templated app]().
+Name the project startup_namer. You’ll be modifying this starter
+app to create the finished app.
+
+### Update main.dart
+
+You'll update two files during this codelab. In the next step
+you'll update the pubspec file, `pubspec.yaml`, which manages
+the app's assets.
+You'll mostly be editing **lib/main.dart**, where the Dart code lives.
+
+<aside class="alert alert-success" markdown="1">
+<i class="fa fa-lightbulb-o"> </i> **Tip:**
+When pasting code into your app, indentation can
+become skewed. You can fix this in Android Studio by right-clicking the
+Dart code and selecting **Reformat Code with dartfmt**.
+Or, at the command line, you can use
+[dartfmt](https://github.com/dart-lang/dart_style).
+</aside>
+
+<ol markdown="1">
+
+<li markdown="1">
+Replace all of the code in lib/main.dart with the following,
+which displays "Hello World" in the center of the screen.
+
+<!-- skip -->
+{% prettify dart %}
+import 'package:flutter/material.dart';
+
+void main() => runApp(new MyApp());
+
+class MyApp extends StatelessWidget {
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'Welcome to Flutter',
+      home: new Scaffold(
+        appBar: new AppBar(
+          title: new Text('Welcome to Flutter'),
+        ),
+        body: new Center(child: new Text('Hello World')),
+      ),
+    );
+  }
+}
+{% endprettify %}
+</li>
+
+<li markdown="1">
+Run the app. You should see the following screen.
+
+<center><img src="images/hello-world-screenshot.png" alt="screenshot of hello world app"></center>
+</li>
+
+</ol>
+
+### Observations
+
+<ul markdown="1">
+<li markdown="1"> This example creates a Material app.
+</li>
+<li markdown="1"> The main method specifies fat arrow (`=>`) notation,
+     which is short hand used for one-line functions or methods.
+</li>
+<li markdown="1"> The app extends StatelessWidget which makes the app itself a
+     widget. In Flutter, most everything is a widget, including
+     alignment, padding, and layout.
+</li>
+<li markdown="1"> The Scaffold widget, from the Material library,
+     provides a default app bar, title, and a body property that
+     holds the widget tree for the home screen. The widget subtree
+     can be quite complex.
+</li>
+<li markdown="1"> The widget tree for this example consists of a Center widget
+     containing a Text child widget. The Center widget aligns its
+     widget subtree to the center of the screen.
+</li>
+
+<li markdown="1"> A widget’s main job is to provide a `build` function
```

Comment:
The `build` method probably should be introduced before the concept of `widget tree`, since the `build` method is where you specify your UI as a tree of widgets.

Reply:
Done